Kudu.Core/Infrastructure/OperationClient.cs

```diff
@@ -35,7 +35,17 @@ public OperationClient(ITracer tracer)
         public async Task<HttpResponseMessage> PostAsync<T>(string path, T content = default(T))
         {
             var jwt = System.Environment.GetEnvironmentVariable(Constants.X_MS_SITE_RESTRICTED_JWT);
+            if (String.IsNullOrEmpty(jwt))
```

Comment:
This part is orthogonal to _Sync Triggers needs to respect Disabled functions_, right?